packages/plugin-core/src/commands/ShowPreview.ts

```diff
@@ -94,31 +102,104 @@ export const extractNoteIdFromHref = (data: {
   return out;
 };
 
-enum LinkType {
+export enum LinkType {
   WIKI = "WIKI",
+  ASSET = "ASSET",
   WEBSITE = "WEBSITE",
   MARKDOWN = "MARKDOWN",
   UNKNOWN = "UNKNOWN",
 }
 
 const classifyLink = ({ href }: NoteViewMessage["data"]): LinkType => {
   if (href && href.startsWith("vscode-webview")) {
-    return LinkType.WIKI;
-  }
-  if (href && (href.startsWith("http://") || href.startsWith("https://"))) {
+    if (href.includes("/assets/")) {
```

Comment:
Assets are referenced via `/assets/`,  `assets/` and `./assets/`. The above would miss the second case. 

There's also an edge case where we could have someone with a vault named assets at which point the cross vault link would look like the following: `dendron://assets/{wiki-link}`

I forget if the vault name shows up as part of the `href`. I like to say no but good to double check

Reply:
Good call out didn't know `dendron://assets/{wiki-link}` was a thing. 

On the different asset formatting it appears by the time it gets to `href` they are all formatted the same added manual testing for that as well. Let me know if you meant some different way to reference the assets than the ones that are added in the manual test:

```
* [dummy-pdf.pdf](assets/dummy-pdf.pdf) 
* [dummy-pdf.pdf](./assets/dummy-pdf.pdf) 
* [dummy-pdf.pdf](/assets/dummy-pdf.pdf) 
```

Reply:
miscommunication here - when i say they could be a vault for assets, I meant someone can create a vault and name it `assets`. 

in that case, you would have `assets` appear in the url but actually not be an asset. so `[[ dendron://assets/foo]]` should open the note foo. if the vault name doesn't show up in the `href` then the point should be moot and we don't need to handle it